src/ui/src/builder/journal/BuilderJournalEntry.vue

```diff
@@ -67,16 +72,26 @@ const { formattedDate, formattedTime } = useDateTimeFormatter(dateObj, {
 	height: 100%;
 	display: flex;
 	flex-direction: row;
+	align-items: center;
 	gap: 12px;
 }
 
-.BuilderJournalEntry__summary__component {
+.BuilderJournalEntry__summary__content {
 	display: flex;
-	align-items: center;
+	flex-direction: column;
+	gap: 0px;
 	justify-content: center;
+}
+
+.BuilderJournalEntry__summary__component {
 	font-weight: 500;
 }
 
+.BuilderJournalEntry__summary__instanceType {
+	font-size: 0.75rem;
```

Comment:
We try to avoid `rem` and use `px` instead. I know we still have legacy `rem` in some places, but let's try to get rid of them.
```suggestion
	font-size: 12px;
```